test/Bridges/utilities.jl

```diff
@@ -34,3 +29,63 @@ function test_delete_bridge(
         test_noc(m, noc...)
     end
 end
+function test_delete_bridged_variable(
+    m::MOIB.AbstractBridgeOptimizer, vi::MOI.VariableIndex, S::Type,
+    nvars::Int, nocs::Tuple; used_bridges = 1, num_bridged = 1, used_constraints = 1)
+    function num_bridges()
+        return count(bridge -> true, values(MOIB.Variable.bridges(m)))
+    end
+    start_num_bridges = num_bridges()
+    @test MOI.get(m, MOI.NumberOfVariables()) == nvars
+    @test length(MOI.get(m, MOI.ListOfVariableIndices())) == nvars
+    if S != MOI.Reals
+        F = S <: MOI.AbstractScalarSet ? MOI.SingleVariable : MOI.VectorOfVariables
+        test_noc(m, F, S, num_bridged)
```

Comment:
What is a noc? This abbreviation saves typing but doesn't make the code easier to read (http://www.juliaopt.org/JuMP.jl/dev/style/#Abbreviations-1).